opennms-doc/guide-admin/src/asciidoc/text/operation/timeseries/configuration.adoc

```diff
@@ -5,16 +5,16 @@
 
 ===== Enabling Timeseries Integration Layer
 
-_{opennms-product-name}_ can be configured to use the Timeseries Integration Layer by setting the following property in in `${OPENNMS_HOME}/etc/opennms.properties`:
+_{opennms-product-name}_ can be configured to use the Timeseries Integration Layer by setting the following property in `${OPENNMS_HOME}/etc/opennms.properties`:
 
 [source]
 ----
 org.opennms.timeseries.strategy=integration
 ----
 
-After activating the Timeseries Integration Layer an actual implementation needs to be started.
-This is done via Karaf.
-Here is an example on how to activate the im memory plugin:
+After activating the _Timeseries Integration Layer_, you need to start an actual implementation.
+Do this via Karaf.
+Here is an example of how to activate the im memory plugin:
```

Comment:
Just to check: do we need to define "im" in "im" memory plugin, or will people know?